sdk/documenttranslation/azure-ai-documenttranslation/README.md

```diff
@@ -1,35 +1,402 @@
+    )
+]
+```
+
+> Note: the target_url for each target language must be unique.
+
+See the service documentation for all [supported languages][supported_languages].
+
+### Return value
+
+There are primarily two types of return values when checking on the result of a translation job - `JobStatusResult` and `DocumentStatusResult`.
+* A `JobStatusResult` will contain the details of the entire job, such as it's status, ID, any errors, and status summaries of the documents in the job.
+* A `DocumentStatusResult` will contain the details of an individual document, such as it's status, translation progress, any errors,
+and the URLs to the source document and translated document.
+
+## Examples
+
+The following section provides several code snippets covering some of the most common Document Translation tasks, including:
+
+* [Translate your documents](#translate-your-documents "Translate Your Documents")
+* [Check status on individual documents](#check-status-on-individual-documents "Check Status On Individual Documents")
+* [List translation jobs](#list-translation-jobs "List Translation Jobs")
+
+### Translate your documents
+Translate the documents in your source container to the target containers, using custom glossaries.
+
+```python
+from azure.core.credentials import AzureKeyCredential
+from azure.ai.documenttranslation import DocumentTranslationClient, DocumentTranslationInput, TranslationTarget
+
+endpoint = "https://<resource-name>.cognitiveservices.azure.com/"
+credential = AzureKeyCredential("<api_key>")
+source_container_sas_url_en = "<sas-url-en>"
+target_container_sas_url_es = "<sas-url-es>"
+target_container_sas_url_fr = "<sas-url-fr>"
+
+document_translation_client = DocumentTranslationClient(endpoint, credential)
+
+glossaries = ["<glossary-url-A>", "<glossary-url-B>"]
+
+job = document_translation_client.create_translation_job(
+    [
+        DocumentTranslationInput(
+            source_url=source_container_sas_url_en,
+            targets=[
+                TranslationTarget(target_url=target_container_sas_url_es, glossaries=glossaries, language_code="es"),
+                TranslationTarget(target_url=target_container_sas_url_fr, glossaries=glossaries, language_code="fr"),
+            ],
+        )
+    ]
+)  # type: JobStatusResult
+
+job_result = document_translation_client.wait_until_done(job.id)  # type: JobStatusResult
+
+print("Job created on: {}".format(job_result.created_on))
+print("Job last updated on: {}".format(job_result.last_updated_on))
+print("Total number of translations on documents: {}".format(job_result.documents_total_count))
+
+print("Of total documents...")
+print("{} failed".format(job_result.documents_failed_count))
+print("{} succeeded".format(job_result.documents_succeeded_count))
+print("{} in progress".format(job_result.documents_in_progress_count))
+print("{} not yet started".format(job_result.documents_not_yet_started_count))
+print("{} cancelled".format(job_result.documents_cancelled_count))
```

Comment:
do u think it is worth adding this 3 knowing that in the example, at this point, all documents should have been done, and none where cancelled?